### PR TITLE
fix issue #1254 Allow the same context to be used multiple times in configuration

### PR DESCRIPTION
**Pull request by lzhan**
The unit test will not pass until the https://github.com/yahoo/ycb/pull/21 is merged and ycb is updated.

### CODE REVIEW

File: lib/app/addons/rs/config.js [hunk 1]
```diff
@@ -222,11 +222,6 @@ YUI.add('addon-rs-config', function(Y, NAME) {
                         Y.log('missing "settings" in YCB file: ' + path, 'error', NAME);
                         return;
                     }
-                    if (settings[section.settings]) {
-                        Y.log('settings "' + section.settings + '" exists in both \n' + path +
-                                ' and\n' + settings[section.settings], 'error', NAME);
-                        return;
-                    }
                     settings[section.settings] = path;
```
Comment (drewfish) [inline lib/app/addons/rs/config.js]: If we're no longer doing `if (settings[section.settings])` then I think we can drop this assignment (`settings[section.settings] = path`) as well.

Comment (caridy) [inline lib/app/addons/rs/config.js]: yeah, this assignment sounds a little bit weird for me as well.

**Comment by drewfish** [comment]
We might want to hold off on this until a new version of ycb is available, and then update package.json to depend on that.  (Otherwise PR is looking good.)

**Comment by drewfish** [comment]
Hmmm... I wonder if there's some way we can get travis to re-test this (now that ycb@1.1.0 has been released).

**Comment by lzhan** [comment]
The last travis run is with ycb 1.1.0. The unit test is fine. Func tests failed on node10 with dashboard app. I don't think it is related to this pr.

**Comment by drewfish** [comment]
+1

**Pull request merged**